src/goto-symex/goto_symex_state.cpp

```diff
@@ -330,13 +330,9 @@ void goto_symex_statet::rename(exprt &expr, const namespacet &ns)
   {
     // we never rename function symbols
     if(as_const(expr).type().id() == ID_code)
-    {
       rename<level>(expr.type(), to_symbol_expr(expr).get_identifier(), ns);
-      return;
-    }
-
-    expr=ssa_exprt(expr);
-    rename<level>(expr, ns);
+    else
+      expr = rename<level>(ssa_exprt(expr), ns);
```

Comment:
Nit pick: you otherwise aim to use `{...}` in constructor calls, so do `ssa_exprt{expr}` here as well.